mlir/lib/Dialect/Tosa/Transforms/TosaFolders.cpp

```diff
@@ -1005,6 +1005,38 @@ struct TosaFoldConstantAdd : public TosaFoldConstantBinary<TosaFoldConstantAdd,
   }
 };
 
+struct TosaFoldConstantSub : public TosaFoldConstantBinary<TosaFoldConstantSub, SubOp> {
+  using TosaFoldConstantBinary<TosaFoldConstantSub, SubOp>::TosaFoldConstantBinary;
+
+  DenseElementsAttr computeInteger(DenseElementsAttr lhsValues,
+                                   DenseElementsAttr rhsValues,
+                                   PatternRewriter &rewriter, SubOp op) const {
+    bool addOverflowed = false;
```

Comment:
Can you refactor to use `sub` instead of `add`? (`addOverflowed`, `intAdd`, `Addition did overflow`, `floatAdd`)